WORKING_GROUPS.md

```diff
@@ -0,0 +1,236 @@
+
+- Write and make sure that docs are up to date.
+- Create Tutorials and Workshops to teach concepts about IPFS and the Distributed Web.
+- Develop the front facing Webpages of each project (ipfs.io, libp2p.io, multiformats.io and ipld.io)
+- Ensure that public forums (discuss.ipfs.io, irc, github, etc) have adequate moderation and responsive support.
+- Ensure that the code of conduct is followed and provide appropriate support or response when violations occur.
+
+#### Internationalisation and Localization (i18n and l10n)
+
+**Coordination**: <n/a>
+
+The Internationalisation and Localization Working Group create the platform for community members to collaborate in their own language.
+
+**Responsibilities include**:
+
+- Translation of the documentation and specification.
+- Monitor social media in the multiple languages supported.
+- Promote speaking engagements in multiple languages.
+- Create subteams to tackle the translation of each language.
+
+### Ongoing Efforts to Support Specific Uses
+
+**sub-index:**
+- [Integration with Web Browsers](#integration-with-web-browsers)
+- [Large Data Sets](#large-data-sets)
+- [IPFS Cluster](#ipfs-cluster)
+
+#### Integration with Web Browsers
+
+**Coordination**: https://github.com/ipfs/in-web-browsers
+
+The Integration with Web Browsers Working Group designs and implement Browser integrations, Web Extensions, Service Workers and any other strategy that contributes to IPFS being integrated with the Web today.
+
+**Responsibilities include**:
+
+- Browser integrations (Firefox, Chrome, Brave and others).
+- IPFS Web Extension.
+- IPFS Service Worker.
+- IPFS Web Worker.
+- Collaborate with with Browser vendors.
+- Identify blockers and design constraints of each integration, figuring out new ways to go around those limitations.
+- Increase the adoption of the Distributed Web by making it easy for users to access it.
+- Define specifications for address schemes and advocate for those schemes to be adopted.
+
+Tracking: https://github.com/ipfs/in-web-browsers
+
+#### Large Data Sets
+
+**Coordination**: <n/a>
+
+The Large Data Sets Working Group is dedicated to create the tooling and the testbed necessary to make sure that IPFS is capable of handling large datasets and that the workflows match or improve on what are the expectations from current industry standards.
+
+**Responsibilities include**:
+
+- Interact directly with organizations that use IPFS for large datasets.
+- ipfs-pack.
+- Data Together initiative.
+
+#### IPFS Cluster
+
+**Coordination**: https://github.com/ipfs/ipfs-cluster
+
+The IPFS Cluster Working Group is the team implementing IPFS Cluster.
+
+**Responsibilities include**:
+
+- Design and Implement IPFS Cluster.
+
+## Other endeavours not yet classified as a working group
+
+Areas of Work that haven't been formalized as a Working Group.
+
+The following bodies of work haven't been structure into a working group.
+
+- InterPlanetary TestLab
+- IPFS Infrastructure
```

Comment:
@lgierth would you see IPFS Infrastructure as a working group (as in, something that the community will be interested in participating, that there will be a team and multiple endeavors) or more as a single project?